<launch>
 <arg name="x_0" default="10"/>
 <arg name="y_0" default="10"/>
 <arg name="x_1" default="10"/>
 <arg name="y_1" default="15"/>

 <node pkg="map_server" type="map_server" name="map_server" args="$(find map_transform)/maps/maze7.png 0.1" respawn="false" > 
    <param name="frame_id" value="/map" />
 </node>
	
 <group ns="robot_0">

    <node pkg="map_transform" type="visibility" name="visibility" output="screen">   
      <param name="defl" value="80" />
      <param name="infl" value="2" />
      <param name="debug" value="false" />
      <param name="ground_truth" value="false" />
	<param name="tf_prefix" value ="/robot_0" /> 
      <remap from="/robot_0/visibility/map" to="/map"/>
    </node>

    <node pkg="tf" type="static_transform_publisher" name="fake_localize_0" args="$(arg x_0) $(arg y_0) 0 0 0 0 /map /robot_0/base_link 10"/>

 </group>

 <node name="planner" pkg="map_transform" type="planSens" output="screen">
       <param name="defl" value="80" />
       <param name="infl" value="2" />
       <remap from="/robot_0/v_map" to="/robot_0/visibility/v_map"/>
       <remap from="/robot_0/e_map" to="/robot_0/visibility/e_map"/>
       <remap from="/planner/graph" to="/robot_0/visibility/graph"/>
 </node>

 <node name="rviz" pkg="rviz" type="rviz" args="-d $(find map_transform)/launch/planning.rviz" />

</launch>
